cves/kernel/CVE-2016-6187.yml

```diff
@@ -1,479 +1,497 @@
+    expressions" is fine, and security phrases like "invalid write" are fine to
+    keep too.
+
+    Your target audience is people just like you before you took any course in
+    security
+description: |
+    "A flaw in the Linux kernel's AppArmor security module could allow a local attacker to crash the system or gain privileges. The apparmor_setprocattr() function did not check the size of data passed to it from proc_pid_attr_write(), which handles writes to /proc filesystem attribute files. This was exposed when proc_pid_attr_write() started using memdup_user() instead of allocating a page of memory, removing implicit size restrictions but also causing a stack overflow. By passing overly large input, an attacker could exploit this to execute malicious code in the kernel. The vulnerability was fixed by adding proper size validation in apparmor_setprocattr(). The flaw showed how missing bounds checks could lead to privilege escalation or denial of service."
 bounty_instructions: |
-  If you came across any indications that a bounty was paid out for this
-  vulnerability, fill it out here. Or correct it if the information already here
-  was wrong. Otherwise, leave it blank.
+    If you came across any indications that a bounty was paid out for this
+    vulnerability, fill it out here. Or correct it if the information already here
+    was wrong. Otherwise, leave it blank.
 bounty:
-  amt:
-  announced:
-  url:
+    amt:
+    announced:
+    url:
 reviews: []
 bugs_instructions: |
-  What bugs are involved in this vulnerability?
+    What bugs are involved in this vulnerability?
 
-  Please list bug IDs to https://bugzilla.kernel.org/
+    Please list bug IDs to https://bugzilla.kernel.org/
 
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
+    Bug ID's can appear in several places:
+      * Mentioned in commit messages
+      * Mentioned in mailing list discussions
+      * References from NVD entry
+      * Various other places
 bugs: []
 fixes_instructions: |
-  Please put the commit hash in "commit" below.
+    Please put the commit hash in "commit" below.
 
-  This must be a git commit hash from the systemd source repo, a  40-character
-  hexademical string/
+    This must be a git commit hash from the systemd source repo, a  40-character
+    hexademical string/
 
-  Place any notes you would like to make in the notes field.
+    Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
-- commit: 30a46a4647fd1df9cf52e43bf467f0d9265096ca
-  note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, please fact-check that this commit fixes the vulnerability and replace this comment with 'Manually confirmed'
+    - commit: 30a46a4647fd1df9cf52e43bf467f0d9265096ca
+      note: "Manually confirmed"
```

Comment:
I would recommend adding some notes to the commits describing briefly what they address to be considered a fix.